sdfObject/sdfobject-genericlevel.sdf.json

```diff
@@ -0,0 +1,134 @@
+{
+  "info": {
+    "title": "Example BT Mesh Generic Level Model", 
+    "version": "20200710", 
+    "copyright": "Copyright 2020 Michael J. Koster. All rights reserved.", 
+    "license": "https://github.com/one-data-model/oneDM/blob/master/LICENSE"
+  }, 
+  "namespace": {
+    "pg": "https://onedm.org/playground/"
+  }, 
+  "defaultnamespace": "pg", 
+  "sdfObject": {
+    "GenericLevel": {
+      "sdfProperty": {
+        "Level": {
+          "sdfRef": "#/sdfObject/GenericLevel/sdfData/GenericLevelData"
+        }
+      },
+      "sdfAction": {
+        "LevelGet": {
+          "description": "Get the status of the Level server",
+          "sdfData": {
+            "PresentLevel": {
+              "sdfRef": "#/sdfObject/GenericLevel/sdfData/GenericLevelData"
+            },
+            "TargetLevel": {
+              "sdfRef": "#/sdfObject/GenericLevel/sdfData/GenericLevelData"
+            },
+            "RemainingTime": {
+              "sdfRef": "#/sdfObject/GenericLevel/sdfData/TransitionTimeData"
+            }
+          }
+        },
+        "LevelSet": {
+          "description": "Set the level server state with time and delay parameters",
+          "sdfInputData": [
+            "#/sdfObject/GenericLevel/sdfAction/LevelSet/sdfData/Level",
+            "#/sdfObject/GenericLevel/sdfAction/LevelSet/sdfData/TransitionTime",
+            "#/sdfObject/GenericLevel/sdfAction/LevelSet/sdfData/Delay"
+          ],
+          "sdfData": {
+            "Level": {
+                "sdfRef": "#/sdfObject/GenericLevel/sdfData/GenericLevelData"
+            },
+            "TransitionTime": {
+              "sdfRef": "#/sdfObject/GenericLevel/sdfData/TransitionTimeData"
+            },
+            "Delay": {
+              "sdfRef": "#/sdfObject/GenericLevel/sdfData/DelayData"
+            }
+          }
+        },
+        "DeltaSet": {
+          "description": "Set the level server state with time and delay parameters",
+          "sdfInputData": [
+            "#/sdfObject/GenericLevel/sdfAction/DeltaSet/sdfData/DeltaLevel",
+            "#/sdfObject/GenericLevel/sdfAction/DeltaSet/sdfData/TransitionTime",
+            "#/sdfObject/GenericLevel/sdfAction/DeltaSet/sdfData/Delay"
+          ],
+          "sdfData": {
+            "DeltaLevel": {
+              "sdfRef": "#/sdfObject/GenericLevel/sdfData/GenericLevelData"
```

Comment:
There is a subtle difference here. The range of the DeltaLevel parameter is -655535 to +65535 (17 bits), as it must cover going the "full swing" either in positive (upwards) or negative (downwards) direction. GenericLevelData is only -32768 to + 32767.

Reply:
fixed this